base/logging_context.go

```diff
@@ -22,3 +24,11 @@ func (lc *LogContext) addContext(format string) string {
 
 	return format
 }
+
+func FormatBlipContextID(contextID string) string {
+	return "[" + contextID + "]"
+}
+
+func FormatChangeCacheContextID(contextName string) string {
+	return fmt.Sprintf("%s-ChangeCache", contextName)
```

Comment:
More efficient to concatenate strings instead of Sprintf for static things like this, see benchmark:
https://gist.github.com/dtjm/c6ebc86abe7515c988ec#gistcomment-2198686